src/components/Item/ItemMessage/MissingPluginMessage.js

```diff
@@ -0,0 +1,45 @@
+import i18n from '@dhis2/d2-i18n'
+import { Cover, IconWarning24, colors } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { isPrintMode } from '../../../modules/dashboardModes.js'
+import classes from './styles/ItemMessage.module.css'
+
+const MissingPluginMessage = ({ dashboardMode, pluginName }) => {
+    return (
+        <Cover>
+            <div className={classes.messageContent}>
+                <IconWarning24 color={colors.grey500} />
+                <span>
+                    {i18n.t(
+                        'The plugin for rendering this item is not available'
```

Comment:
Should we use a less technical word than "rendering" ?

Reply:
I changed the message to your first suggestion.
As for the usefulness of the message, it does make sense for apps that are not bundled (ie. LL) and you open a dashboard which includes a visualization that needs a particular plugin.